src/t8_forest/t8_forest_vtk.cxx

```diff
@@ -421,9 +408,12 @@ t8_forest_vtk_write_file_via_API (t8_forest_t forest, const char *fileprefix,
   vtkDoubleArray    **dataArrays;
   dataArrays = T8_ALLOC (vtkDoubleArray *, num_data);
 
-  cmesh = t8_forest_get_cmesh (forest);
+  const t8_cmesh_t    cmesh = t8_forest_get_cmesh (forest);
 /* We iterate over all local trees*/
-  for (itree = 0; itree < t8_forest_get_num_local_trees (forest); itree++) {
+  t8_element_t        element;
```

Comment:
When you move the `element` inside of the for-loop that iterates over all elements you can make it const.